packages/invoices/src/OutboundShipment/DetailView/OutboundLineEdit/OutboundLineEditForm.tsx

```diff
@@ -46,6 +46,18 @@ export const OutboundLineEditForm: React.FC<OutboundLineEditFormProps> = ({
     Math.abs(Number(packSizeController.selected?.value || 1));
   const { items } = useOutboundRows();
 
+  const onChangePackSize = (newPackSize: number) => {
+    const previousPackSize = packSizeController.selected?.value || 1;
+    const newQuantity =
+      previousPackSize > newPackSize ? quantity * previousPackSize : quantity;
+
+    packSizeController.setPackSize(newPackSize);
```

Comment:
as requested. If you have `100` quantity in a pack size of `1` then switch to pack size of `10` - the quantity should change to `10` for you.

Reply:
Nice 😁

Reply:
Is this the right behaviour? I'm unsure but I suspect not as what I am reading is that the *unit quantity* should stay the same when the pack size is changed. 

pack size of 1 - unit quantity: 2000
![image](https://user-images.githubusercontent.com/35858975/158341177-f2f08e84-b756-436f-b18c-c18cc0f18b2a.png)

Changed to a pack size of 10 - unit quantity 2000
![image](https://user-images.githubusercontent.com/35858975/158341212-70288c3a-683c-4128-b439-c1804919821c.png)

Changed to a pack size of 50 - unit quantity 200
![image](https://user-images.githubusercontent.com/35858975/158341431-8eda6cb2-eca8-4dce-895b-771321bb0f53.png)

What I observe is:
When the pack size is one, changing to ten:
- `quantity` is `2000`
- `newQuantity` becomes `2000`
- Changes the quantity to `2000 / 10 = 200` with a pack size of `10` a total unit quantity of `2000`

When the pack size is ten, changing to 50:
- `quantity` is `200`
- `newQuantity` becomes `200`
- Changes the quantity to `200/50 = 4` with a pack size of `50`, a total unit quantity of `200`

I'm not confident I understand fully the requirement but if the *unit quantity* should stay the same, then we can simply do:

```
const onChangePackSize = (newPackSize: number) => {
    const newAllocatedQuantity = allocatedQuantity / newPackSize; // some other fancy math safety stuffs
    packSizeController.setPackSize(newPackSize);
    onChangeQuantity(
      newAllocatedQuantity,
      newPackSize === -1 ? null : newPackSize
    );
  };
```

Reply:
thanks!
yes - the intention was to have the unit quantity constant. that's much better 😁